angular-workspace/projects/component-library/src/lib/component-library.module.ts

```diff
@@ -3,7 +3,7 @@ import { NgModule } from "@angular/core";
 import { DIRECTIVES } from "./stencil-generated";
 
 @NgModule({
-  declarations: [...DIRECTIVES],
+  imports: [...DIRECTIVES],
```

Comment:
N.a.v. angular 19.

In Webstorm:

<img width="939" alt="Screenshot 2024-12-10 at 15 31 50" src="https://github.com/user-attachments/assets/a27f50d8-2f97-4065-a732-eabb77df638c">

In de browser:
```
Unexpected "DsoSelectable" found in the "declarations" array of the "StorybookComponentModule" NgModule, "DsoSelectable" is marked as standalone and can't be declared in any NgModule - did you intend to import it instead (by adding it to the "imports" array)?
```

Reply:
Klinkt goed